site.conf

```diff
@@ -79,10 +79,10 @@
           'acead4f6f335c7bea4455935dbcabf86c6c6452b8be89aad11ed705cf7496ea8', -- Marvin
         },
       },
-      test = {
-        name = 'test',
+      rc = {
+        name = 'rc',
         mirrors = {
-          'http://updates.services.ffffm.net/test/sysupgrade',
```

Comment:
ggf wechseln wir mal auf den neuen DL Server :D  
Aber denn teste ich lieber erstmal

Reply:
Kann in einen eigenen PR wenn noch nicht Operational? Das blockt ja nichts an dieser Stelle.